### PR TITLE
Order payment account

### PR DESCRIPTION
**Pull request by robinnedergard**

### CODE REVIEW

File: Payment/Account/Creatable.ts [hunk 1]
```diff
@@ -5,12 +5,17 @@ import { CreatableBase } from "../CreatableBase"
 export interface Creatable extends CreatableBase {
 	type: "account"
 	token: authly.Token
```
Comment (simonmika) [inline Payment/Account/Creatable.ts:7]: Optional? Deprecated?

File: Payment/Account/index.ts [hunk 2]
```diff
@@ -5,11 +5,18 @@ import { Creatable as AccountCreatable } from "./Creatable"
 export interface Account extends Base {
 	type: "account"
 	token: authly.Token
```
Comment (simonmika) [inline Payment/Account/index.ts:7]: Deprecated?

File: Payment/Account/index.ts [hunk 3]
```diff
@@ -5,11 +5,18 @@ import { Creatable as AccountCreatable } from "./Creatable"
...
+	account?: authly.Identifier
 }
```
Comment (simonmika) [inline Payment/Account/index.ts:9]: Retry schedule? Not now but in the next part.

File: Payment/Card/Creatable.ts [hunk 4]
```diff
@@ -5,7 +5,7 @@ import { CreatableBase } from "../CreatableBase"
 
 export interface Creatable extends CreatableBase {
 	type: "card"
-	account?: string
+	account?: authly.Identifier
```
Comment (simonmika) [inline Payment/Card/Creatable.ts:8]: Not in Creatable? When would it be used?

File: Payment/Card/index.ts [hunk 5]
```diff
@@ -6,7 +6,7 @@ import { Creatable as CardCreatable } from "./Creatable"
 export interface Card extends Base {
 	type: "card"
 	card?: authly.Token
-	account?: string
+	account?: string // Old orders exists with account as authly.Token although for new orders account should be of type authly.Identifier.
```
Comment (simonmika) [inline Payment/Card/index.ts:9]: `authly.Token | authly.Identifier` // Token is deprecated.

**Review by simonmika — CHANGES_REQUESTED** [review]

File: Payment/Card/Creatable.ts [hunk 6]
```diff
@@ -5,7 +5,6 @@ import { CreatableBase } from "../CreatableBase"
 
 export interface Creatable extends CreatableBase {
 	type: "card"
-	account?: string
```
Comment (simonmika) [inline Payment/Card/Creatable.ts:8]: Are we certain that nobody uses this?

**Review by simonmika — APPROVED** [review]

File: Payment/Card/Creatable.ts (hunk 6 above)
Comment (robinnedergard) [reply Payment/Card/Creatable.ts:8]: We did in backend, but only after internal conversion between payment type account -> card. I have an update almost ready to change that.

Comment (simonmika) [reply Payment/Card/Creatable.ts:8]: @fredrikbryntesson Feel free to merge and go ahead with this but keep the above question in mind.

Comment (robinnedergard) [reply Payment/Card/Creatable.ts:8]: Checked that the two I know could have used it, does use the correct way already (type: "account"), that will still work. Website information needs updating though.

**Review by fredrikbryntesson — APPROVED** [review]

**Pull request merged**